skimage/segmentation/random_walker_segmentation.py

```diff
@@ -30,7 +30,7 @@ def new_del(self):
             pass
     umfpack.UmfpackContext.__del__ = new_del
     UmfpackContext = umfpack.UmfpackContext()
-except:
+except:  # noqa
```

Comment:
an exception should be specified et leas by `Exception`

Reply:
I agree, and here we can probably do even better. @JDWarner might have some thoughts?

Reply:
These kind of maintenance changes really require 3 devs working according to our policies. Will not work with such large sweeping changes without focused effort ;)